web/src/elements/table/TableSearch.ts

```diff
@@ -16,17 +17,23 @@ import PFToolbar from "@patternfly/patternfly/components/Toolbar/toolbar.css";
 import PFBase from "@patternfly/patternfly/patternfly-base.css";
 
 @customElement("ak-table-search")
-export class TableSearch extends WithLicenseSummary(AKElement) {
-    @property()
-    public value?: string;
+export class TableSearchForm extends WithLicenseSummary(AKElement) {
+    @property({ type: String, reflect: false })
+    public defaultValue?: string;
```

Comment:
This is a really tricky property since the element here is more like a form than a single search input. We also want to avoid communicating that the value is timely after the user interacts with the search field. IMO the name is fitting since the input `defaultValue` property [expresses a similar concept](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/defaultValue)